server/controllers/entities/lib/move_to_wikidata.ts

```diff
@@ -24,9 +27,25 @@ export async function moveInvEntityToWikidata (user: User, invEntityUri: InvEnti
 
   let claims, labels
   if ('labels' in entity) labels = entity.labels
-  if ('claims' in entity) claims = extendInvClaims(entity.claims)
+  if ('claims' in entity) claims = expandInvClaims(entity.claims)
 
-  const { uri: wdEntityUri } = await createWdEntity({ labels, claims, user, isAlreadyValidated: true })
+  const conflictingWdEntities = await resolveExternalIds(claims, {
+    resolveOnWikidata: true,
+    resolveLocally: false,
+    refresh: true,
+  })
+  if (conflictingWdEntities.length > 0) {
```

Comment:
When I create an edition without isbn, then click on 'move to Wikidata' following error pops up :

```
****** Cannot read properties of undefined (reading 'length') ******
TypeError: Cannot read properties of undefined (reading 'length')
    at moveInvEntityToWikidata (server/controllers/entities/lib/move_to_wikidata.ts:37:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async controllerWrapper (server/lib/controller_wrapper.ts:33:22) {
[skip...]
  context: { body: { uri: 'inv:6d495495dee900063c5ee8afc349ed7f' } }
}
-----
PUT /api/entities?action=move-to-wikidata 500 10ms
```

and indeed `resolve_external_ids.ts` line 43:
```
if (propertyValuePairs.length === 0) return
```
returns an `undefined` value

```suggestion
  if (conflictingWdEntities && conflictingWdEntities.length > 0) {
```

Reply:
When i solve the previous bug above, another error pops up (still on edition's "move to wikidata"):

```
****** wd entity creation ******
{
  labels: {},
  claims: {
    'wdt:P1476': [ { value: 'La méthode Google' } ],
    'wdt:P577': [ { value: '2009' } ],
    'wdt:P1104': [ { value: 426 } ],
    'wdt:P2969': [ { value: '5100525' } ],
    'wdt:P31': [ { value: 'wd:Q3331189' } ],
    'wdt:P629': [ { value: 'wd:Q100989307' } ],
    'wdt:P212': [ { value: '978-0-06-170971-5' } ],
    'wdt:P957': [ { value: '0-06-170971-9' } ],
    'wdt:P407': [ { value: 'wd:Q1860' } ]
  }
}
-----
****** 400 ******
Error: invalid monolingualtext value
    at Object.new (inventaire/node_modules/wikibase-edit/lib/error.js:3:17)
    at Object.snakValue (inventaire/node_modules/wikibase-edit/lib/validate.js:76:23)
    at fullClaimBuilder (inventaire/node_modules/wikibase-edit/lib/entity/build_claim.js:52:14)
    at module.exports (inventaire/node_modules/wikibase-edit/lib/entity/build_claim.js:21:12) {
  statusCode: 400,
  context: { property: 'P1476', value: 'La méthode Google' },
  user: { _id: '5785b6c45d065900e48c865e0b00084e', username: 'adamsberg' },
  headers: {
    referer: 'http://localhost:3005/entity/isbn:9780061709715/edit'
  }
}
-----
PUT /api/entities?action=move-to-wikidata 400
```

[edit: when i click on move to wd from a collection editor, then I am asked to add a language claim (aka `This collection can not be moved to Wikidata as it lacks a language claim`, then i add a language claim, then when i click, i have the same error as above `invalid monolingualtext value`)]

Reply:
Some testing was indeed much needed! Fixed in 06939e532